command/node_status.go

```diff
@@ -111,6 +116,7 @@ func (c *NodeStatusCommand) Run(args []string) int {
 		fmt.Sprintf("Datacenter|%s", node.Datacenter),
 		fmt.Sprintf("Drain|%v", node.Drain),
 		fmt.Sprintf("Status|%s", node.Status),
+                fmt.Sprintf("Attributes|%s", strings.Join(attributes, ", ")),
```

Comment:
That's because of the + sign. If you click on 'View', you will see that it lines up.